compiler/rustc_borrowck/src/diagnostics/conflict_errors.rs

```diff
@@ -911,6 +919,9 @@ impl<'cx, 'tcx> MirBorrowckCtxt<'cx, 'tcx> {
         issued_borrow: &BorrowData<'tcx>,
         explanation: BorrowExplanation<'tcx>,
     ) {
+        if matches!(explanation, BorrowExplanation::UsedLaterInLoop(..)) {
+            return;
+        }
```

Comment:
Isn't the test with `used_in_call` sufficient?

Reply:
This is for not providing user a suggestion how to fix the problem when UsedLaterInLoop case.

Reply:
Yes, it's redundant, removed.